index.js

```diff
@@ -0,0 +1,195 @@
+const imageSection = document.getElementById("image-parent");
+const sliderDots = document.getElementById("slides_dots");
+const nextButton = document.querySelector(".next");
+const previousButton = document.querySelector(".prev");
+const slidesContainer = document.getElementById("slides")
+// Array of image URLs
+const arrayOfImages = [
+  "/images/1.jpg",
+  "/images/2.jpg",
+  "/images/3.jpg",
+  "/images/4.jpg",
+  "/images/5.jpg",
+  "/images/6.jpg",
+  "/images/7.jpg",
+  "/images/8.jpg",
+  "/images/9.jpg",
+];
+// Variables to store image references and slide counter
+let allImage;
+let slideCounter = 0;
+
+// Function to add images to the slides
+const addImageInSlides = () => {
```

Comment:
why do you need a function to trigger this functionality.
you can directly initiate it.